addons/mail_bot/static/src/scss/discuss.scss

```diff
@@ -0,0 +1,20 @@
+.o_notification_alert {
+    > span {
+        display: block;
+    }
+    .o_url_style {
+        margin-top: 16px;
+        padding: 8px;
+        width: 75%;
+        position: relative;
+        font-style: italic;
+        text-align: left;
+
+        .o_clipboard_button {
+            padding: 8px;
+            position: absolute;
```

Comment:
I think absolute positioning makes no sense here, as clipboard button is always next to url input?

Reply:
If we remove "position: absolute", UI doesn't seen in proper way.

Reply:
That's because for intended UI you should use flex layout instead